config.go

```diff
@@ -24,6 +26,20 @@ type Config struct {
 	DiscloseIdentity bool
 	Hooks            Hooks
 	nicks            map[string]string // normalizeURL(url) -> nick
+	path             string            // location of loaded config
```

Comment:
This is a good idea. Wonder if we could get rid of the configpath global...

Reply:
Possibly IF I can i'll do so in a "cleanup" PR.